python/src/mas/cli/install/app.py

```diff
@@ -184,6 +184,10 @@ def configMongoDb(self) -> None:
         self.printH1("Configure MongoDb")
         self.promptForString("Install namespace", "mongodb_namespace", default="mongoce")
 
+    def configSpecialCharacters(self):
```

Comment:
What versions of MAS support this ... we should only show this prompt if the MAS version supports the option

Reply:
This change was back-ported to all mas version